blackjax/mcmc/ula.py

```diff
@@ -0,0 +1,38 @@
+"""Public API for Unadjusted Langevin kernels."""
+import jax
+from blackjax.types import PyTree, PRNGKey
+from blackjax.base import SamplingAlgorithm
+
+from typing import Callable, NamedTuple, Tuple
+from blackjax.mcmc.langevin import DiffusionState, overdamped_langevin
+from blackjax.mcmc.gradients import GradientState
+
+__all__ = ["DiffusionState", "ULAInfo", "init", "kernel"]
+
+class UlaInfo(NamedTuple):
+    pass
+
+
+
+def kernel(logprob_fn: Callable, step_size: float):
+    _grad_fn = jax.grad(logprob_fn)
+
+    def grad_fn(state: DiffusionState) -> PyTree:
+        return _grad_fn(state.position)
+
+    integrator = overdamped_langevin(grad_fn)
+
+    def one_step(rng_key, state):
+        info = None
+        return integrator(rng_key, state, step_size), info
+
+    return one_step
+
+
+
+def init(position: PyTree, logprob_fn: Callable) -> DiffusionState:
```

Comment:
We've added `init` before `kernel` in the other kernel files, we should follow this convention here too.